src/utils/getTimeDifference.ts

```diff
@@ -1,3 +1,6 @@
-import { formatDistance } from 'date-fns'
+import dayjs from 'dayjs'
+import relativeTime from 'dayjs/plugin/relativeTime'
 
-export const getTimeDifference = (createdAt: string): string => formatDistance(createdAt, new Date())
+dayjs.extend(relativeTime)
+
+export const getTimeDifference = (createdAt: string): string => dayjs().to(dayjs(createdAt))
```

Comment:
This utility function is hard to comprehend. I think it is better to write some comments along with some example of both input and output for this function. 

I also believe that this function is more suited inside the `hoc` that I suggest to create. We don't need utilities for such computations which can simply reside inside the `hoc` or hooks.